### PR TITLE
also build and install FreeImagePlus

### PR DESCRIPTION
**Pull request by fiesh**
Install the C++ wrapper of freeimage as well

### CODE REVIEW

**Comment by starius** [comment]
Could you replace too similar calls to `$(MAKE)` with `for target in freeimage freeimageplus; do ...`?

**Comment by fiesh** [comment]
Good idea, thanks, done and rebased.

**Comment by starius** [comment]
It builds for me. :+1: 

This package installs `freeimage.pc` file for C.
Could you also install `freeimageplus.pc` for C++, please?

This package has a test of compilation and linking for C version.
Could you add the a test for C++ wrapper, please?

**Comment by fiesh** [comment]
OK, put the test and the pkgconfig in -- and fixed the building, it was incorrect as the test revealed.  also rebased. Thanks for the feedback.

**Comment by starius** [comment]
It builds successfully here.

File: src/freeimage.mk [hunk 1]
```diff
@@ -23,14 +23,24 @@ define $(PKG)_BUILD
         AR='$(TARGET)-ar' \
         RC='$(TARGET)-windres' \
         FREEIMAGE_LIBRARY_TYPE=STATIC \
-        TARGET=freeimage
+        TARGET=freeimage; \
```
Comment (tonytheodore) [inline src/freeimage.mk]: Minor nitpick, we don't want to continue here.

Comment (fiesh) [inline src/freeimage.mk]: Thanks, true, I updated and rebased!

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**